parastell/parastell.py

```diff
@@ -622,10 +660,16 @@ def parastell():
     
     if args.ivb or args.magnets:
         dagmc_export = all_data['dagmc_export']
+        stellarator.build_cubit_model(
+            **(filter_kwargs(dagmc_export, build_cubit_model_allowed_kwargs)))
         stellarator.export_dagmc(
             export_dir=args.export_dir,
-            **dagmc_export
+            **(filter_kwargs(dagmc_export, export_allowed_kwargs))
         )
+        cub5_export = all_data['cub5_export']
+        if cub5_export['export_cub5']:
+            stellarator.export_cub5(export_dir=args.export_dir,
+                                    **cub5_export)
```

Comment:
Use proper formatting.
```suggestion
            stellarator.export_cub5(
                export_dir=args.export_dir, **cub5_export
            )
```